package.json

```diff
@@ -74,6 +74,7 @@
     "react-codemirror2": "^5.1.0",
     "react-fout-stager": "^3.0.0",
     "react-router-dom": "^4.3.1",
+    "react-test-renderer": "^16.11.0",
```

Comment:
Does it work without this? We didn't have this library before.

Reply:
Yes, we can safely delete this